CONTRIBUTING.md

```diff
@@ -149,3 +149,23 @@ in general try to follow them.
   ensure that the exporter provides [zPages](https://opencensus.io/zpages/) and a standard set of metrics.
 - `replace` statements in `go.mod` files can be automatically inserted by running `make crosslink`. For more information
   on the `crosslink` tool see the README [here](https://github.com/open-telemetry/opentelemetry-go-build-tools/tree/main/crosslink).
+
+## Becoming a Code Owner
+
+A Code Owner is responsible for a component within Collector Contrib, as indicated by the [CODEOWNERS file](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS). That responsibility includes maintaining the component, responding to issues, and reviewing pull requests.
+
+Sometimes a component may be in need of a new or additional Code Owner.  A few reasons this situation may arise would be:
+- The component was never assigned a Code Owner.
+- A previous Code Owner stepped down.
+- An existing Code Owner has become unresponsive. See [unmaintained stability status](https://github.com/open-telemetry/opentelemetry-collector#unmaintained).
+- The existing Code Owners are actively looking for new Code Owners to help.
+
+If you would like to help and become a Code Owner you must meet the following requirements:
+
+1. Have contributed to the OpenTelemetry Project, preferably either Collector Core or Collector Contrib. Best case scenario would be contributing to the component directly.
+2. Have attended the Collector SIG meeting.
```

Comment:
Is there a reason to have these requirements differ from the OpenTelemetry contribution guidelines for membership? I *think* it would make sense for a contributor to the project and code owner to be at least a member of the organization, see: https://github.com/open-telemetry/community/blob/main/community-membership.md#member

Reply:
I am not opposed to making Project Membership a requirement for component code ownership, but I don't think we've required it in the past.  Never too late to start.

Reply:
If we don't, we should as project membership is a requirement for assigning issues to users who haven't interacted with the issue.

Reply:
I wasn't aware that project member was a pre-req for issue assignment. In that case I definitely think we should require membership. 

SIG attendance is a tricky one I agree. I added it because I thought there was another place in our docs that had something like "attended 2 SIG meetings" but now I can't find it. In my mind the intended goal of attendance is proof of participation in the community. There are other ways to do that of course. Overall, measuring participation is hard.

Reply:
Updated requirements based on SIG meeting discussion.  The only mandatory requirement is community memership.